### PR TITLE
docs(asset-search): rewrite stories in new format

### PR DESCRIPTION
**Pull request by amevck**

### CODE REVIEW

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-544.gearboxjs.preview.cogniteapp.com

File: src/components/AssetSearch/stories/helper.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import { assetsList } from '../../../mocks';
+import { MockCogniteClient } from '../../../utils/mockSdk';
+import { ClientSDKProvider } from '../../ClientSDKProvider';
+import { AssetSearchStyles } from '../AssetSearch';
+
+class CogniteClient extends MockCogniteClient {
+  assets: any = {
+    // @ts-ignore
+    list: (scope: AssetListScope) => {
+      action('assets.list')(scope);
```
Comment (mrtraser) [inline src/components/AssetSearch/stories/helper.tsx]: we don't need this `action` here

File: src/components/AssetSearch/stories/examples.story.mdx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+      advancedSearch={true} 
+      rootAssetSelect={true} 
+      styles={styles}
+    />
+  </Story>
+</Preview>
+
+## Handle Search Results
+
+<!-- TODO show dynamic search results here -->
```
Comment (mrtraser) [inline src/components/AssetSearch/stories/examples.story.mdx]: why this comment here?

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-544.gearboxjs.preview.cogniteapp.com

**Comment by mrtraser** [comment]
Screenshot 2019-12-18 at 09 52 43

Can we have a higher story section to avoid scrollbar appearing when dropdown list is rendered?

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-544.gearboxjs.preview.cogniteapp.com

**Comment by mrtraser** [comment]
it still has this issue with height for basic example
Screenshot 2019-12-19 at 14 14 37

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-544.gearboxjs.preview.cogniteapp.com

**Pull request merged**